fuel/datasets/youtube_audio.py

```diff
@@ -0,0 +1,10 @@
+from fuel.datasets.hdf5 import H5PYDataset
+from fuel.utils import find_in_data_path
+
+
+class YouTubeAudio(H5PYDataset):
+    def __init__(self, youtube_id, **kwargs):
+        super(YouTubeAudio, self).__init__(
+            file_or_path=find_in_data_path('{}.hdf5'.format(youtube_id)),
+            which_sets=('train',), **kwargs
+        )
```

Comment:
Yup.

> The closing brace/bracket/parenthesis on multi-line constructs may either line up under the first non-whitespace character of the last line of list, as in:

> ```
my_list = [
    1, 2, 3,
    4, 5, 6,
    ]
result = some_function_that_takes_arguments(
    'a', 'b', 'c',
    'd', 'e', 'f',
    )
```
> or it may be lined up under the first character of the line that starts the multi-line construct, as in:
```
my_list = [
    1, 2, 3,
    4, 5, 6,
]
result = some_function_that_takes_arguments(
    'a', 'b', 'c',
    'd', 'e', 'f',
)
```